Default (OSX).sublime-mousemap

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "button": "button1", 
+        "modifiers": ["ctrl"], 
```

Comment:
I used the same mouse bindings as SublimeCodeIntel ([osx mousemap](https://github.com/SublimeCodeIntel/SublimeCodeIntel/blob/master/Default%20%28OSX%29.sublime-mousemap)), because it is a very common package (TOP 25) and this should feel similar. However I have never used osx and don't know how the buttons are used and which buttons are available.